packages/foreman/rubygem-thread_safe/rubygem-thread_safe.spec

```diff
@@ -24,6 +24,8 @@ BuildArch: noarch
 Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
 # end specfile generated dependencies
 
+Obsoletes: tfm-ror52-rubygem-%{gem_name} <= 0.3.6
```

Comment:
I think it should at least obsolete 0.3.6-4?

Reply:
I copied this from all other obsoletes. Perhaps they all need to be modified?